beesdoo_shift_attendance/tests/test_beesdoo_shift.py

```diff
@@ -20,7 +20,7 @@ def setUp(self):
         self.pre_filled_task_type_id = (
             self.env["ir.config_parameter"]
             .sudo()
-            .get_param("beesdoo_shift.pre_filled_task_type_id")
+            .get_param("beesdoo_shift_attendance.pre_filled_task_type_id")
```

Comment:
How come this didn't break the test before?

Reply:
because of the aforementioned `default` arg.